invenio_app_rdm/config.py

```diff
@@ -632,6 +632,9 @@ def files_rest_permission_factory(obj, action):
 }
 """Deposit file upload quota """
 
+APP_RDM_DEPOSIT_FORM_DECIMAL_FILE_SIZES = True
```

Comment:
```suggestion
APP_RDM_DISPLAY_DECIMAL_FILE_SIZES = True
```

Comment: naming-wise since this is also applied on the record landing page. Non-blocking of course :)

Reply:
true, i forgot to change the name to reflect that fact!